### PR TITLE
Adds stuff for fallout.

### PR DESCRIPTION
**Pull request by Nienhaus**
rga9gnx
h3z5g9z
botiwta

I added an eyebot which is pretty much beepsky that can't attack and a "Dauntless Bot" aka Mister Gutsy.
mlblnhn

### CODE REVIEW

**Comment by Nienhaus** [comment]
Almost forgot, Enclave stuff from @Snapwalk

**Comment by Nienhaus** [comment]
I did it :D

File: code/modules/urist/items/food.dm [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+	trash = /obj/item/trash/urist/bubblegum
+	filling_color = "#ffe98c"
+
+	New()
+		..()
+		reagents.add_reagent("sugar", 2)
+
+/obj/item/weapon/reagent_containers/food/snacks/urist/cram
+	name = "cram"
+	desc = "Carm! You know, food, we think."
```
Comment (scrdest) [inline code/modules/urist/items/food.dm]: misspell

File: code/modules/urist/mob/robots.dm [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+	icon = 'icons/urist/events/fallout.dmi'
+	idcheck = 0
+	auto_patrol = 1
+
+/mob/living/simple_animal/hostile/retaliate/malf_drone/urist
+	name = "bot"
+	desc = "Bot."
+	icon = 'icons/urist/events/fallout.dmi'
+
+/mob/living/simple_animal/hostile/retaliate/malf_drone/urist/robot
```
Comment (Glloyd) [inline code/modules/urist/mob/robots.dm]: Drones slowly die IIRC. If that's what you want, then cool.

**Comment by Glloyd** [comment]
Thanks Nien!

**Pull request merged**